contracts/payloads/IGP105/PayloadIGP105.sol

```diff
@@ -0,0 +1,585 @@
+            TEAM_MULTISIG_2,
+            true
+        );
+    }
+
+    // @notice Action 4: Set wstUSR launch limits and dust limits
+    function action4() internal isActionSkippable(4) {
+        // Set launch limits for existing WSTUSR/STABLE vaults (T1 vaults)
+        {
+            address wstUSR_USDC_VAULT = getVaultAddress(110);
+            
+            // [TYPE 1] WSTUSR/USDC vault - Launch limits
+            VaultConfig memory VAULT_wstUSR_USDC = VaultConfig({
+                vault: wstUSR_USDC_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: wstUSR_ADDRESS,
+                borrowToken: USDC_ADDRESS,
+                baseWithdrawalLimitInUSD: 6_000_000, // $6M
+                baseBorrowLimitInUSD: 6_000_000, // $6M
+                maxBorrowLimitInUSD: 12_000_000 // $12M
+            });
+            
+            setVaultLimits(VAULT_wstUSR_USDC);
+            VAULT_FACTORY.setVaultAuth(wstUSR_USDC_VAULT, TEAM_MULTISIG, false);
+        }
+        
+        {
+            address wstUSR_USDT_VAULT = getVaultAddress(111);
+            
+            // [TYPE 1] WSTUSR/USDT vault - Launch limits
+            VaultConfig memory VAULT_wstUSR_USDT = VaultConfig({
+                vault: wstUSR_USDT_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: wstUSR_ADDRESS,
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 6_000_000, // $6M
+                baseBorrowLimitInUSD: 6_000_000, // $6M
+                maxBorrowLimitInUSD: 12_000_000 // $12M
+            });
+            
+            setVaultLimits(VAULT_wstUSR_USDT);
+            VAULT_FACTORY.setVaultAuth(wstUSR_USDT_VAULT, TEAM_MULTISIG, false);
+        }
+        
+        {
+            address wstUSR_GHO_VAULT = getVaultAddress(112);
+            
+            // [TYPE 1] WSTUSR/GHO vault - Dust limits
+            VaultConfig memory VAULT_wstUSR_GHO = VaultConfig({
+                vault: wstUSR_GHO_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: wstUSR_ADDRESS,
+                borrowToken: GHO_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 20_000 // $20k
+            });
+            
+            setVaultLimits(VAULT_wstUSR_GHO);
+            VAULT_FACTORY.setVaultAuth(wstUSR_GHO_VAULT, TEAM_MULTISIG, true);
+        }
+        
+        // Set dust limits for WSTUSR T3 vaults with USDC-USDT smart debt
+        {
+            address wstUSR_USDC_USDT_VAULT = getVaultAddress(143);
+            
+            // [TYPE 2] WSTUSR-USDC<>USDC-USDT vault - Dust limits
+            VaultConfig memory VAULT_wstUSR_USDC_USDT = VaultConfig({
+                vault: wstUSR_USDC_USDT_VAULT,
+                vaultType: VAULT_TYPE.TYPE_2,
+                supplyToken: address(0), // Set at DEX level
```

Comment:
other way around, borrow token must be set at dex level, this are the T3 vaults 

we must set supply token WSTUSR at LL and borrow side at the respective dex